public/js/angular/filters.js

```diff
@@ -47,8 +44,8 @@ angular
   .filter('generateGravatar', function () {
     return function (hash) {
       var
-      default_avatar = "https%3A%2F%2Fstuff.webmaker.org%2Favatars%2Fwebmaker-avatar-44x44.png",
-        default_size = 44;
-      return "https://secure.gravatar.com/avatar/" + hash + "?s=" + default_size + "&d=" + default_avatar;
+      DEFAULT_AVATAR = 'https%3A%2F%2Fstuff.webmaker.org%2Favatars%2Fwebmaker-avatar-44x44.png',
```

Comment:
in that case: the updated PR still uses all caps snake_case, rather than camelCase.